bin/chatai

```diff
@@ -1,22 +1,34 @@
 #!/usr/bin/env ruby
 
-require "dotenv"
-require "openai"
-require "reline"
-require "word_wrap"
+require 'dotenv'
+require 'openai'
+require 'reline'
+require 'word_wrap'
 
 Dotenv.load(".env", "~/.chatai.env")
+SAVE_HISTORY = ENV['SAVE_HISTORY'] == 'true'
+DEV_MODE = ENV['DEV_MODE'] == 'true'
+
+LANGUAGE_TO_EXTENSION = {
+  "ruby" => "rb",
+  "python" => "py",
+  "html" => "html",
+  "javascript" => "js",
+  "java" => "java",
+  "c" => "c",
+  "css" => "css"
+}
 
 OpenAI.configure do |config|
-  config.access_token = ENV.fetch("OPENAI_API_KEY") do |name|
-    print "OpenAI API Key: "
-    gets.chomp
+  config.access_token = ENV.fetch("OPENAI_API_KEY") do 
+    puts "Don't forget to add your Api Key on the .env file"
+    exit 1
   end
 end
 
 def chat(client, input)
   parameters = {
-    model: ENV.fetch("OPENAI_MODEL", "gtp-3.5-turbo"),
+    model: ENV.fetch("OPENAI_MODEL", "gpt-3.5-turbo"),
```

Comment:
Would you mind creating a separate PR to fix this typo? I can merge that quickly :)